pypeman/message.py

```diff
@@ -89,7 +89,12 @@ def add_context(self, key, msg):
 
     def to_dict(self):
         """
-        Convert the current message object to a dict. Payload is pickled.
+        Convert the current message object to a dict, that can be converted 
+            to a json.
+        Therefore the payload and all the context is pickled and base64 encoded.
+
+        Warning: the returned dict cannot be converted to json if meta contains
+            objects, that connot be dumped as json.
```

Comment:
*connot* ;-)

Reply:
done in 72864c95246dea5272825240c66657f0168da487

Reply:
can close?